### DISCUSSION TITLE
Login to a worker

### DISCUSSION THREAD

**Discussion by oliviermollier** [jhuckaby/Cronicle#543] [Q&A] [answered]
Hello, do you know if it's possible to log on the webpage of a worker? I saw the web screen is here but i can't log in.

**Comment by jhuckaby** [comment, marked as answer]
I'm so sorry, but this is not currently possible.  While worker servers technically have a web server, they cannot serve up the web app.  Only primary or backup servers can do this.

I agree it's weird that they can even serve up the page.  I will look into disabling this, or serving up some kind of error page that explains what is going on.
